OnlTools/Jevp/StJevpBuilders/fstBuilder.h

```diff
@@ -0,0 +1,651 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "JevpBuilder.h"
+#include "DAQ_READER/daqReader.h"
```

Comment:
Try to forward declare instead

```suggestion
class daqReader;
```